innative/lexer.cpp

```diff
@@ -534,7 +551,7 @@ void innative::TokenizeWAT(Queue<WatToken>& tokens, const char* s, const char* e
           tokens.Push(WatToken{ kh_val(tokenhash, iter), begin, line, column });
         else
         {
-          uint8_t op = GetInstruction(ref);
+          uint16_t op = GetInstruction(ref);
```

Comment:
Why was this changed to uint16_t? Is it capturing all possible instruction bytes? Does that work properly? If we are trying to capture all possible instruction bytes, maybe this type should be defined somewhere else so it can be derived from the current max instruction size.

Reply:
Yeah, the hashmap that GetInstruction reads from already contains the 2-byte versions of all the opcodes, with opcode[0] in the lower 8 bits and the high 8 bits set to 0 for single byte instructions. Using a typedef might be a good idea, not sure what kind of refactoring that'd require but I can go through it